### DISCUSSION TITLE
Proper way to use this project

### DISCUSSION THREAD

**Discussion by joeybab3** [PHPAuth/PHPAuth#533] [Q&A]
Huge fan of this project, I've been using it to handle login for a few personal projects. I'm not 100% sure if I'm using it right and I see that you guys are looking for help with some examples. Would a few examples for things like registration page/login/login protected page be helpful/useful for you guys? I would love some feedback on the proper way to use it and I feel like that could be mutually beneficial.

**Comment by JapCLoN** [comment]
Hello @joeybab3,

Yes, we've wanted to make an example repo for a very long time, sadly most of us are very busy.
If you feel like you have some good starting example to start off with, I'll be happy to create the repo so you can submit some PRs for it and we will take it from there, where we can give feedbacks and others can submit PRs or issues to create even better starting exmaples on how to use XYZ functions.

**Reply by joeybab3** [reply]
Perfect! I understand how hard it is to maintain open source when you feel like there are not enough hours in the day, its essentially an unpaid job but I am happy to contribute some of my work and see if it helps you guys as a starting point at least
